src/main/java/be/syntra/devshop/DevshopFront/configuration/WebConfig.java

```diff
@@ -34,5 +40,37 @@ RestTemplate restTemplate(RestTemplateBuilder builder) {
                 .interceptors(Collections.singleton(interceptor))
                 .build();
     }
+
+    @Bean
+    public CartDto getCurrentCart() {
+        List<Product> productList = new ArrayList<>();
+        return CartDto.builder()
+                .cartCreationDateTime(LocalDateTime.now())
+                .finalizedCart(false)
+                .activeCart(true)
+                .paidCart(false)
+                .products(productList)
+                .build();
+    }
+
+    // todo: DEV-015 might not create the SearchModelDto here
+
+    @Bean
+    public SearchDto getSearchModelDto() {
```

Comment:
rename method to match the returntype -> getSearchDto

Reply:
Nu de SearchDto een SearchModel is geworden, misschien best de methode wel aanpassen naar getSearchModel